Iterators and Closures/Standard Library Types/Iterable Collection/task.md

```diff
@@ -0,0 +1,20 @@
+## Iterable Collection
+
+Make the code compile by filling in the `???`s
+
+When performing operations on elements within a collection, iterators are essential.
+This module helps you get familiar with the structure of using an iterator and
+how to go through elements within an iterable collection.
+
+Check out the hints if you're stuck!
+
+<div class="hint">We need to apply something to the collection <code>my_fav_fruits</code> before we start to go through
+it. What could that be? Take a look at the struct definition for a vector for inspiration:
```

Comment:
it. What could that be? Take a look at the struct vector definition for inspiration: